src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <a class="logo" href="#home">
+        <img
+          class="logoImg"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="start">
+            <a class = "links is-active" href="#apple">Apple</a>
+          </li>
+          <li>
```

Comment:
add one empty line between same level elements
```suggestion
          </li>
          
          <li>
```